scm_v3c/applications/sensor_adc/svd.c

```diff
@@ -0,0 +1,120 @@
+
+// ONLY FOR 2X2 MATRICES
+
+#include "svd.h"
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "matrix.h"
+
+matrix_t mtx;
```

Comment:
In general, don't abbreviate anything: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules